dashboard/metrics_utils.py

```diff
@@ -0,0 +1,13 @@
+import os
+import psutil
+from ray.dashboard import k8s_utils
+
+# Are we in a K8s pod?
+IN_KUBERNETES_POD = "KUBERNETES_SERVICE_HOST" in os.environ
+
+
+def get_cpu_percent(in_k8s: bool):
+    if in_k8s:
```

Comment:
nit; 

```
if IN_KUBERNETES_POD:
    return k8_utils...
```
?

Reply:
removed the whole file - this code is for system wide usage, we just want proc level usage.